src/ontology/edits/oeo-social.omn

```diff
@@ -592,13 +592,17 @@ Class: OEO_00010117
 Class: OEO_00010123
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Effort sharing is a policy instrument that sets a common emission reduction goal for a number of countries. Each country has a national reduction goal. Under- or overachievement of that goal can be compensated by trading emission certificates.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Effort sharing is a regulatory instrument that sets a common emission reduction goal for a number of countries. Each country has a national reduction goal. Under- or overachievement of that goal can be compensated by trading emission certificates."@en,
```

Comment:
As we have more flexibilities than trading the emission certificates under Effort Sharing, how about adjusting slightly to:

"Effort sharing is a regulatory instrument that sets a common emission reduction goal for a number of countries. Each country has a national reduction goal. Under- or overachievement of that goal can be compensated by various flexibilities given in the regulation.

Reply:
Thanks for the rephrasing. I will implement.